### ISSUE TITLE
Problema al intentar registrarse en evento, utilizando Boton GitHub

### ISSUE THREAD

**Issue by mvpsdvlpr** [sysarmy/2024.nerdear.la#10]
- ESP Ver:
Intentando registrarme al evento, en la URL https://reg.nerdear.la/?utm_source=hero.
Utilizando el botón, para poder loguearme con mi usuario de GitHub.
Aparece un error al momento de querer autenticarse uno mediante el uso de esta funcionalidad.
Dejo ejemplo visual respectro al problema identificado.

- ENG Ver:
Trying to register for the event, at the URL https://reg.nerdear.la/?utm_source=hero.
Using the button, to be able to log in with my GitHub user.
An error appears when trying to authenticate using this functionality.
I leave a visual example regarding the identified problem

- Visual example / Ejemplo visual:

![2024-04-08 17-49-21](https://github.com/sysarmy/2024.nerdear.la/assets/65572029/80eeabe5-7cfd-42e1-9694-0cd59b4631eb)

**Comment by Nachichuri** [comment]
Hola! Gracias por el reporte.

El warning de third party cookie no debería interferir con el oauth... Cuando tengas 5 minutos, me harías el favor de probar el registro con GitHub desde una ventana de incógnito y usando la misma cuenta por favor??

Gracias!!

**Comment by mvpsdvlpr** [comment]
Hola @Nachichuri, realice la prueba que comentaste… en modo incógnito, el problema no se presentó y me envió de forma inmediata al login de GitHub

![image](https://github.com/sysarmy/2024.nerdear.la/assets/65572029/e811a091-3b2b-485b-8534-71c097cedcce)

**Comment by Nachichuri** [comment]
Pero te dejó autenticarte con github? O cuando tratas de loguearte te tira el mismo error?

**Comment by mvpsdvlpr** [comment]
Claro, a eso me referia con mi comentario de **"me envió de forma inmediata al login de GitHub"**, no ocurrio ningun problema.
Lo que no estaba ocurriendo, desde la pagina de nerdearla, sin utilizar el modo incognito, esta poder ser enviado al formulario de login que existe en GitHub y arrojaba error, como expuse en el inicio de este issue.

**Comment by Nachichuri** [comment]
Ok genial, vamos a revisar entonces el workflow de oauth para github, muchas gracias!

**Comment by mvpsdvlpr** [comment]
@Nachichuri no hay problema, un gusto haber sido de ayuda 😁
